### PR TITLE
Backend/feature/profile model

### PR DESCRIPTION
**Pull request by bartuinceQR**
Added token creation upon login, and profile information.

### CODE REVIEW

File: backend/workhub/controllers/user_controller.js [hunk 1]
```diff
@@ -78,4 +106,47 @@ exports.login = function(req,res){
...
+			where: {id : user_id}
+		}]
+	}).then(info => {
+		if (info){
+			firstName = info.user.firstName;
+			lastName = info.user.lastName;
+			desc = info.description;
+			rating = info.rating;
+
+			console.log(desc);
```
Comment (ergunsh) [inline backend/workhub/controllers/user_controller.js]: I think, we can remove these logs; wdyt?

**Review by ergunsh — COMMENTED** [review]
I think, this is good to go; however, this branch is behind of `develop`  very much. Let me rebase this to `develop` and get latest changes. Then, we can merge this.

File: backend/workhub/controllers/user_controller.js (hunk 1 above)
Comment (bartuinceQR) [reply backend/workhub/controllers/user_controller.js]: yeah, not really necessary at this point

Comment (ergunsh) [reply backend/workhub/controllers/user_controller.js]:
```suggestion

```

File: backend/workhub/controllers/user_controller.js [hunk 2]
```diff
@@ -59,8 +63,32 @@ exports.login = function(req,res){
      		bcrypt.compare(req.body.password, users.password, function (err, result) {
 				
      			if(result == true){
```
Comment (ergunsh) [inline backend/workhub/controllers/user_controller.js]: We can refactor this if part of the code.

This is same as `if(result) {`

File: backend/workhub/controllers/user_controller.js [hunk 3]
```diff
@@ -59,8 +63,32 @@ exports.login = function(req,res){
...
+     					}
+     				})		
+
      				res.status(200).send({
-						msg: "Logged in successfully."
+						msg: "Logged in successfully.",
+						token: token
             		})
       			}
      	 		else if(result == false){
```
Comment (ergunsh) [inline backend/workhub/controllers/user_controller.js]: only `else {` is sufficient.

**Review by ergunsh — APPROVED** [review]
LGTM with a few improvements for backend team 😅

**Pull request merged**